packages/@sfx/dom-events-plugin/test/unit/common/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { AssertTypesEqual, expect } from '../../utils';
+import {
+  DomEventsPlugin as DomEventsPluginExport,
+  DomEventsPluginOptions as DomEventsPluginOptionsExport,
+  DomEventsPluginExposedValue as DomEventsPluginExposedValueExport,
+} from '../../../src';
+import DomEventsPlugin from '../../../src/dom-events-plugin';
+import { DomEventsPluginOptions, DomEventsPluginExposedValue } from '../../../src/dom-events-plugin';
+
+describe('Entry point', () => {
+  it('should export DomEventsPlugin ', () => {
+    expect(DomEventsPluginExport).to.equal(DomEventsPlugin);
+  });
+
+  it('should export the DomEventsPluginOptions interface', () => {
+    const test: AssertTypesEqual<DomEventsPluginOptions , DomEventsPluginOptionsExport> = true;
```

Comment:
Reorder for consistency with `expect()`:
 
```suggestion
    const test: AssertTypesEqual<DomEventsPluginOptionsExport, DomEventsPluginOptions> = true;
```